### PR TITLE
Update GKE with Ubuntu

### LINKED ISSUE #813: Update GKE User Document with Ubuntu
## Is this a BUG REPORT or FEATURE REQUEST? Bug Report
 
**What happened**: Doc Enhancement with Ubuntu

**What you expected to happen**: User document must have ubuntu details.

### PR DESCRIPTION
**Pull request by mahebbar**
**What this PR does / why we need it**: GKE with Ubuntu was not documented

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that isue when PR gets merged)*: fixes #813

### CODE REVIEW

File: documentation/source/install/cloud_solutions.rst [hunk 1]
```diff
@@ -257,76 +257,94 @@ Minimum requirements for container cluster are as follows:
...
-    sudo service open-iscsi restart
-
-Verify that iSCSI is configured
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-1. Check that initiator name is configured and iSCSI service is running using the following commands.
+a. Check that initiator name is configured.
 ::
 
     sudo cat /etc/iscsi/initiatorname.iscsi
```
Comment (yudaykiran) [inline documentation/source/install/cloud_solutions.rst]: sudo cat /etc/iscsi/initiatorname.iscsi = ~$ sudo cat /etc/iscsi/initiatorname.iscsi

To differentiate between command and output.

File: documentation/source/install/cloud_solutions.rst [hunk 2]
```diff
@@ -257,76 +257,94 @@ Minimum requirements for container cluster are as follows:
...
+    ## If you change the InitiatorName, existing access control lists
+    ## may reject this initiator.  The InitiatorName must be unique
+    ## for each iSCSI initiator.  Do NOT duplicate iSCSI InitiatorNames.
+    InitiatorName=iqn.1993-08.org.debian:01:6277ea61267f
+    
+
+b. Check if iSCSI service is running using the following commands.
+:: 
+
+  sudo service open-iscsi status
```
Comment (yudaykiran) [inline documentation/source/install/cloud_solutions.rst]: sudo cat /etc/iscsi/initiatorname.iscsi = ~$ sudo cat /etc/iscsi/initiatorname.iscsi

To differentiate between command and output.

File: documentation/source/install/cloud_solutions.rst [hunk 3]
```diff
@@ -257,76 +257,94 @@ Minimum requirements for container cluster are as follows:
...
+Apply OpenEBS Operator and add related OpenEBS Storage Classes, that can then be used by developers and applications using the following command.
+::
+
+    kubectl apply -f openebs-operator.yaml
     kubectl apply -f openebs-storageclasses.yaml
 
 **Note:**
 
-The persistent storage is carved out from the space available on the nodes (default host directory : */var/openebs*). Development is in progress to provide administrator with additional options of consuming the storage (as outlined in *openebs-config.yaml*). These are slated to work hand-in-hand with the local storage manager of Kubernetes that is due in Kubernetes 1.7/1.8.
+Persistent storage is created from the space available on the nodes (default host directory : */var/openebs*). Development is in progress to provide administrator with additional options of consuming the storage (as outlined in *openebs-config.yaml*). These are scheduled to work hand-in-hand with the local storage manager of Kubernetes that is due in Kubernetes 1.7/1.8.
```
Comment (yudaykiran) [inline documentation/source/install/cloud_solutions.rst]: due in Kubernetes 1.7/1.8.

Kubernetes 1.7 and 1.8 are already released. This sentence needs to change.

File: documentation/source/install/cloud_solutions.rst [hunk 4]
```diff
@@ -343,6 +361,8 @@ The *kubectl apply -f demo/jupyter/demo-jupyter-openebs.yaml* command creates th
 
 * Launch a Jupyter Server, with the specified notebook file from github (kubectl get deployments)
 * Create an OpenEBS Volume and mounts to the Jupyter Server Pod (/mnt/data) (kubectl get pvc) (kubectl get pv) (kubectl get pods)
-* Expose the Jupyter Server to external world via the http://NodeIP:32424 (NodeIP is any of the nodes external IP) (kubectl get pods)
```
Comment (yudaykiran) [inline documentation/source/install/cloud_solutions.rst]: I guess this step was missed?

File: documentation/source/install/cloud_solutions.rst (hunk 2 above)
Comment (mahebbar) [reply documentation/source/install/cloud_solutions.rst]: Done!

File: documentation/source/install/cloud_solutions.rst (hunk 1 above)
Comment (mahebbar) [reply documentation/source/install/cloud_solutions.rst]: Done!

File: documentation/source/install/cloud_solutions.rst (hunk 3 above)
Comment (mahebbar) [reply documentation/source/install/cloud_solutions.rst]: Done!

**Pull request merged**